src/api/Themer.ts

```diff
@@ -0,0 +1,83 @@
+import { Theme } from "../entities";
+import { Constants, getByName, setAMOLEDThemeEnabledBypass } from "../metro";
+import { readdir } from "../native/fs";
+import { Logger } from "../utils";
+import { THEME_DIRECTORY } from "../utils/constants";
+import { after } from "../utils/patcher";
+
+const logger = new Logger("Themer");
+export const themes = {} as Record<string, Theme>;
+export let currentTheme: Theme;
+
+enum ThemeType {
+    DARK,
+    LIGHT,
+    AMOLED
+}
+
+export function setTheme(theme: Theme) {
+    window.Aliucord.settings.set("theme", theme.name);
+    currentTheme = theme;
+    applyTheme();
+}
+
+export function applyTheme() {
+    if (currentTheme === undefined) return;
```

Comment:
what about resetting theme? don't we need to store the original color map

Reply:
tried that before, even if i restore original color map discord just breaks its theme somehow